### PR TITLE
Remove config migration from AcMailer 4.5 and earlier to AcMailer 5.0

### PR DESCRIPTION
**Pull request by Scyt8l3**

### CODE REVIEW

File: config/module.config.php [hunk 1]
```diff
@@ -2,12 +2,11 @@
 
 namespace AcMailer;
 
+use AcMailer\Service\MailService;
```
Comment (acelaya) [inline config/module.config.php]: What did you add this for?

**Comment by acelaya** [comment]
I was considering doing this myself, so it is very welcome :-)

File: config/module.config.php (hunk 1 above)
Comment (Scyt8l3) [reply config/module.config.php]: My ide has added automatically. I can remove it. No problem

Comment (acelaya) [reply config/module.config.php]: Looks good. I'll probably test it and merge it this weekend. Thanks :-)

**Pull request merged**